app/services/product_service.rb

```diff
@@ -0,0 +1,7 @@
+class ProductService
+
+  def get_service(product)
+    Service.find_or_create_by(product_id: product.id)
```

Comment:
FYI, the comments with a :question: are more for myself, to clarify when I read other parts of the code. Nothing to do here (yet) :smiley:

Reply:
this was just to avoid having the class calling it to have knowledge of how the database works. I could also just place this in the model.

What do you think?

Reply:
> What do you think?

For me, this is just extra complication without benefit. I would just call `find_or_create_by` directly in the model.